Frontend/ui-driver/src/Screens/Types.purs

```diff
@@ -260,8 +261,21 @@ type DriverProfileScreenData = {
   otpLimit :: Int,
   otpBackAlternateNumber :: Maybe String,
   gender :: Maybe String,
-  languageList :: Array CheckBoxOptions
-}
+  languageList :: Array CheckBoxOptions,
+  rcsData :: Array RcData,
+  rcsExceptFirst :: Array RcData,
+  renderingRc :: RcData,
```

Comment:
`rcsExceptFirst` and `renderingRc` can also be renamed to something more understandable